[workers] Add language around multiple package managers (#11838)
cloudflare/cloudflare-docs: content/workers/wrangler/commands.md

wrangler <COMMAND> <SUBCOMMAND> [PARAMETERS] [OPTIONS]
```

Since we recommend installing Wrangler [locally in your project](/workers/wrangler/install-and-update/) (rather than globally), the way to run Wrangler will depend on your specific setup & package manager. 

{{<tabs labels="npm | yarn | pnpm">}}
{{<tab label="npm" default="true">}}

```sh
$ npx wrangler <COMMAND> <SUBCOMMAND> [PARAMETERS] [OPTIONS]
```

{{</tab>}}
{{<tab label="yarn">}}

```sh
$ yarn wrangler <COMMAND> <SUBCOMMAND> [PARAMETERS] [OPTIONS]
```

{{</tab>}}
{{<tab label="pnpm">}}

```sh
$ pnpm wrangler <COMMAND> <SUBCOMMAND> [PARAMETERS] [OPTIONS]
```

{{</tab>}}
{{</tabs>}}

> The easiest way to run Wrangler commands that you use often is to add them as scripts in your project's `package.json` file:

Reviewer: ```suggestion
You can add Wrangler commands that you use often as scripts in your project's `package.json` file:
```
